scripts/romanisim-make-stack

```diff
@@ -0,0 +1,302 @@
+                             'usage).')
+
+    # Number of objects to simulate
+    parser.add_argument('-n', '--nobj',
+                        type=int,
+                        default=1000,
+                        help='Specify number of objects to simulate.')
+
+    # Create script making argument
+    parser.add_argument('-x', "--prefix",
+                        type=str,
+                        metavar='roman_simulated',
+                        default='roman_simulated',
+                        help='Filename prefix for simulated files.')
+
+    # Persistence
+    parser.add_argument('-p', '--previous',
+                        action='store_true',
+                        help=(
+                            'Enable persistence modeling of previous files for exposures after the first in each SCA.'))
+
+    # Random number seed
+    parser.add_argument('-r', '--rng_seed',
+                        type=int,
+                        default=None,
+                        help='Random number seed (int)')
+
+    # SCA detector identifier number
+    parser.add_argument('-s', '--sca',
+                        type=int,
+                        default=0,
+                        help='SCA to simulate')
+
+    # Use CRDS for distortion maps
+    parser.add_argument('-u', '--usecrds',
+                        action='store_true',
+                        help='Use CRDS for distortion map')
+
+    # Use CRDS stored PSFs
+    parser.add_argument('-w', '--webbpsf',
+                        action='store_true',
+                        help='Use webbpsf for PSFs.')
+
+    args = parser.parse_args()
+
+    log.info('Starting simulation...')
+    log.warning("romanisim is under active development.  Its output has "
+                "not been formally validated; only limited testing has been "
+                "performed.  For this reason, use of romanisim for "
+                "preparation of ROSES proposals is not advised.  Other "
+                "packages like galsim's roman package or STIPS may better "
+                "serve such purposes.")
+
+    # Set up metadata
+    metadata = deepcopy(default_parameters_dictionary)
+
+    # Obtain pointing information from input file
+    pointings = Table.read(args.input_file_name, comment="#", delimiter=" ")
+
+    # Obtain RA & Dec pointing extremes from the catalog
+    max_ra = max(pointings["RA"])
+    min_ra = min(pointings["RA"])
+    ra_center = (max_ra + min_ra) / 2
+    metadata['wcsinfo']['ra_ref'] = ra_center
+    max_dec = max(pointings["DEC"])
+    min_dec = min(pointings["DEC"])
+    dec_center = (max_dec + min_dec) / 2
+    metadata['wcsinfo']['dec_ref'] = dec_center
+    ra_diff = max_ra - min_ra
+    dec_diff = max_dec - min_dec
+
+    if args.debug:
+        print(f"max_ra = {max_ra}")
+        print(f"min_ra = {min_ra}")
+        print(f"ra_diff = {ra_diff}")
+        print(f"max_dec = {max_dec}")
+        print(f"min_dec = {min_dec}")
+        print(f"dec_diff = {dec_diff}")
```

Comment:
I don't use it a lot, but we have a log object, and these could be log.debug(...) without a special new debug argument, right?

I worry a bit about these ra/dec box calculations.  If someone wants to observe two images near the celestial pole, this ends up generating a catalog for the full sky?

I think to my sensibilities I just would not support "we'll make up a simple catalog for you" mode for this script.  The present default will do 1k objects over the full "survey area".  The current 1k / SCA is already pretty anemic, but 1k per field of view or larger will be very few objects.

Reply:
You are right about the RA/DEC, though fixing that should be too difficult. Regarding the default objects, yeah, I was thinking that should be set to an object density, and then multiplied by the area to get an object total. If you like this idea, I can make a new ticket / issue to tackle them and remove them from this PR.

I will shift the debug stuff to logging.

Reply:
Yeah, I think that would be the right way to go.  But more broadly, I think we want to make people specify a scene / catalog.  I think it's kind of nice having a "no scene" feature for romanisim-make-image just as the simplest possible entry point for people, but I don't think we want to try to make that feature more broadly.  I feel like if you've gotten to the point where you're simulating a full observational program it makes sense to make you specify a real catalog.

So I'd advocate just for removing that code and feature from the "stack" mode rather than trying to build out everything we'd want.

Reply:
Okay, I will remove nobj, make a catalog file required, and gut that stuff out of the code.